src/sentry/workflow_engine/models/data_condition.py

```diff
@@ -107,9 +110,14 @@ def evaluate_value(self, value: T) -> DataConditionResult:
             op = condition_ops.get(condition, None)
 
         if condition_handler is not None:
-            result = condition_handler.evaluate_value(value, self.comparison, self.condition)
+            result = condition_handler.evaluate_value(
+                value, self.comparison, self.condition, **kwargs
+            )
         elif op is not None:
-            result = op(cast(Any, value), self.comparison)
+            if len(signature(op).parameters) == 1:
+                result = op(cast(Any, value)) == self.comparison
```

Comment:
`operator.truth` and some other operators like `abs` take in 1 argument, the other operators take in 2. if you want to compare the ones with a single argument to the comparison then we need special logic

Reply:
yeah, but do we need to add the truth operator? could we just say `Value === True` instead of adding the truth operator? we also don't have a use case for `abs`, since the goal of data conditions is to compare a dynamic value (value) to a static value (comparison), it feels a little weird to have a single evaluation.